test/specs/mount.spec.js

```diff
@@ -52,9 +52,9 @@ describeRunIf(process.env.TEST_ENV !== 'node', 'mount', () => {
     const wrapper = mount(ComponentWithProps, { propsData: { prop1 }})
     expect(wrapper.vm).to.be.an('object')
     if (wrapper.vm.$props) {
-      expect(wrapper.vm.$props.prop1).to.equal(prop1)
+      expect(wrapper.vm.$props.prop1).to.deep.equal(prop1)
```

Comment:
I confirmed the processing of Vue.js, but it was the same value. So, I thought it would be better not to use `deep`.

> Another solution I could think of was to throw an error and tell the user to call setProps with a new object.

I think that is a good way. But I want to use the same object.
I'll look into a way to fix again.